core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -1132,6 +1133,16 @@ public void format() throws IOException {
     }
   }
 
+  @Override
+  public String toString() {
+    return MoreObjects.toStringHelper(this)
+        .add("Journal Path", mConf.getPath())
```

Comment:
This returns a `File` type, do you need to convert it to string by getting the file path?

Reply:
There are a toString method implemented in File class.